Семинар_5q/Task_5-1.py

```diff
@@ -0,0 +1,12 @@
+a = int(input("Введите число А: "))
+b = int(input("Введите число B: "))
+
+
+def exp(a, b):
+    e = a
+    if b == 0:
+        return 1
+    return e * exp(a, b - 1)
```

Comment:
Не совсем понимаю, зачем вам ещё одна переменная.
```suggestion
    if b == 0:
        return 1
    return a * exp(a, b - 1)
```